templates/http-py/metadata/spin-template.toml

```diff
@@ -1,15 +1,13 @@
 manifest_version = "1"
-id = "http-py"
+id = "http-py-simple"
```

Comment:
Consider keeping the template and directory names in sync (the template system doesn't care but people trying to view/work on the template might).

Reply:
should this be suffixed with `simple` or just `http-py`. We could potentially add a template for streaming HTTP.

Reply:
For others we use `http-<lang>` on its own, but that does date from "when there was only one way."  If we want to start a new convention then we can (and migrate other languages over time), although if so I'd prefer a more descriptive word than "simple."  My inclination would be to stick to `http-py` for now.

(Also we currently have a limitation that if someone upgrades a template repo it doesn't delete ones that no longer exist - it just overwrites where names are the same.  So if we had `http-py-something` then py2wasm users would still have a `http-py` sitting there TEMPTING them.  We should improve this somewhat.)